### PR TITLE
Modify 'Install mod... ' in context menu to install above current mod.

### LINKED ISSUE #1369: "Install mod here" or "Install mod at this position" - this is crucial, please implement.
**Is your feature request related to a problem? Please describe.**
Yes my feature request is related to a problem. The problem is when I install a mod it gets put at the bottom and the window scrolls to it's position. Now I have to either drag it to the position I want or send it to a prio or to a separator, because the chances that I want it at the bottom are astronomically tiny, unless I'm just putting a completly new setup together and I install all mods in the right order from the very beginning, which is very unlikely.
These actions in themselves take more time and clicks than the mod installation.
This makes installing mods on a medium or long setup very cumbersome and it's slower than it could be.

**Describe the solution you'd like**
2 possible solutions I see.

1: If the user right clicks in the left panel and uses the "Install mod" menu then the mod installed this way should be put at the position where he clicked. Just below the mod he highlighted with his click. So if he right clicked at a mod that is at priority 30, the newly installed mod should be put at prio 31. This way he can skip the very cumbersome  part of sending the mod to this position from the bottom.

2nd: If you don't want to alter the "Install mod" behavior, put in a new menu, something like "Install mod here" or "Install mod at this position".

**Describe alternatives you've considered**
There is no alternative I can think of. Suffering more and wasting time and putting more clicks into my old and tired fingers.

**Additional context**
I'm a big fan by the way. MO is great, has been forever. You guys are great.
If you guys having trouble implementing this little qol feature, you can always ask the magus who did Collapsible Separators. He must be like totally hella smart. He will know what to do.
Thank you for everything!

### PR DESCRIPTION
**Pull request by Holt59**
Closes #1369

### CODE REVIEW

File: src/modlistcontextmenu.cpp [hunk 1]
```diff
@@ -24,12 +24,14 @@ void ModListGlobalContextMenu::populate(OrganizerCore& core, ModListView* view,
...
   clear();
 
-  addAction(tr("Install Mod..."), [=]() { view->actions().installMod(); });
 
   auto modIndex = index.data(ModList::IndexRole);
   if (modIndex.isValid() && view->sortColumn() == ModList::COL_PRIORITY) {
     auto info = ModInfo::getByIndex(modIndex.toInt());
     if (!info->isBackup()) {
+
+      addAction(tr("Install mod above..."), [=]() { view->actions().installMod("", index); });
```
Comment (Al12rs) [inline src/modlistcontextmenu.cpp:33]: Why no inside option like for empty mod?

Comment (Holt59) [reply src/modlistcontextmenu.cpp:33]: Because no one asked for it (yet) :D

Comment (Holt59) [reply src/modlistcontextmenu.cpp]: Done.

File: src/modlistviewactions.h [hunk 2]
```diff
@@ -31,9 +31,10 @@ class ModListViewActions : public QObject
     PluginListView* pluginView,
     QObject* nxmReceiver);
 
-  // install the mod from the given archive
+  // install the mod from the given archive before "above" the mod with
```
Comment (Al12rs) [inline src/modlistviewactions.h:34]: Comment is outdated now that it also considers separators and sorting order

Comment (Holt59) [reply src/modlistviewactions.h]: Fixed.

**Review by Al12rs — APPROVED** [review]

**Pull request merged**